okhttp/src/main/kotlin/okhttp3/ExperimentalOkHttpApi.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2024 Block, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3
+
+/**
+ * Marks declarations that are experimental and subject to change without following SemVer
+ * conventions. Both binary and source-incompatible changes are possible, including complete removal
+ * of the experimental API.
+ *
+ * Do not use these APIs in modules that may be executed using a version of OkHttp different from
```

Comment:
Do not use in published libraries without also marking experimental?

Reply:
Even better: Do not use these APIs in published libraries.